### PR TITLE
Logging for Azure SDK for .NET

### PR DESCRIPTION
**Pull request by dsindona**
New article (making Cam the author). @CamSoper please review and do what you will.

[Staging link](https://review.docs.microsoft.com/en-us/dotnet/azure/sdk/logging?branch=pr-en-us-17514)

### CODE REVIEW

File: docs/azure/sdk/logging.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Response headers
+- Error information, when applicable
+
+For request and response content:
+
+- Content stream as text or bytes depending on the Content-Type header.
+     > [!NOTE}
+     > Content logging is disabled by default. To enable it, set `Diagnostics.IsLoggingContentEnabled` to `true` in `ClientOptions`.
+
+Event logs are output usually at one of these three levels:
```
Comment (dsindona) [inline docs/azure/sdk/logging.md]: Pavel suggested adding sample output. Thoughts?

**Comment by CamSoper** [comment]
@BillWagner I self-assigned this for review, but I don't know what labels to put on it. Could you please label it for me?

File: docs/azure/sdk/logging.md (hunk 1 above)
Comment (CamSoper) [reply docs/azure/sdk/logging.md]: I'm not opposed if you have some. I wouldn't make it a huge priority, though.

File: docs/azure/sdk/TOC.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@
       items:
         - name: Authentication
           href: authentication.md
+        - name: Logging
+          href: logging.md
```
Comment (CamSoper) [inline docs/azure/sdk/TOC.yml]: Could you please also add this to the "Concept" group in index.yml?

File: docs/azure/sdk/logging.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+# Logging with the Azure SDK for .NET
+
+The [Azure SDK](https://azure.microsoft.com/downloads/) for .NET client libraries includes the ability to log client library operations. This allows you to monitor I/O requests and responses that client libraries are making to Azure services. Typically, the logs are used to debug or diagnose communication issues. This article describes three approaches to enable logging with the Azure SDK for .NET:
+
+- Log to the console window
+- Log to .NET diagnostics traces
+- Configure custom logging
+
+> [!NOTE]
+> This article applies to client libraries that use the most recent versions of the Azure SDK for .NET. To see if a library is supported, refer to the list of [Azure SDK latest releases](https://azure.github.io/azure-sdk/releases/latest/js.html). If your application is using an older version of the Azure SDK client libraries, refer to specific instructions in the applicable service documentation.
```
Comment (CamSoper) [inline docs/azure/sdk/logging.md]: I really like the phrasing of this. I think I'd call it out with a [!IMPORTANT] instead though.

File: docs/azure/sdk/logging.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+Event logs are output usually at one of these three levels:
+
+- Informational for request and response events
+- Warning for errors
+- Verbose for detailed messages and content logging
+
+## Enable logging with built-in methods
+
+The Azure SDK for .NET client libraries log events to Event Tracing for Windows (ETW) via the [`EventSource` class](/dotnet/api/system.diagnostics.tracing.eventsource), which is standard for the .NET framework. Event sources allow you to use structured logging in your application code with a minimal performance overhead. To gain access to these event logs, you need to register event listeners.
```
Comment (CamSoper) [inline docs/azure/sdk/logging.md]: > The Azure SDK for .NET client libraries log events to Event Tracing for Windows (ETW) via the [`EventSource` class](/dotnet/api/system.diagnostics.tracing.eventsource), ~~which is standard for the .NET framework~~ which is typical for .NET.

The use of "framework" there opens a can of worms we don't want to open. 🙂

**Review by CamSoper — CHANGES_REQUESTED** [review]
LGTM. If you could, please make those couple changes at your leisure and I'll merge it.

Thanks again, Dana!

File: docs/azure/sdk/logging.md (hunk 1 above)
Comment (dsindona) [reply docs/azure/sdk/logging.md]: I don't have any now...

**Review by CamSoper — APPROVED** [review]

**Pull request merged**